src/main/java/net/vfyjxf/nechar/NechConfig.java

```diff
@@ -147,6 +150,23 @@ public static void loadConfig(File configFile) {
         neiAllowedLanguages = config
             .get("nei", "allowedLanguages", new String[] { "zh_CN", "zh_TW" }, "List of languages PinIn is used for")
             .getStringList();
+        EnableIgnoreComma = config
+            .get("search", "EnableIgnoreComma", false, "Set to true to enable ignore comma between numbers")
+            .getBoolean();
+        EnableVoltageSpecialSearchName = config
+            .get(
+                "search",
+                "EnableVoltageSpecialSearchName",
+                false,
+                "!Slow! Set to true to search Voltage names separately for item names")
+            .getBoolean();
+        EnableVoltageSpecialSearchTooltips = config
+            .get(
+                "search",
+                "EnableVoltageSpecialSearchTooltips",
```

Comment:
These should be lowercase (other places will need updates)
```suggestion
        enableIgnoreComma = config
            .get("search", "enableIgnoreComma", false, "Set to true to enable ignore comma between numbers")
            .getBoolean();
        enableVoltageSpecialSearchName = config
            .get(
                "search",
                "enableVoltageSpecialSearchName",
                false,
                "!Slow! Set to true to search Voltage names separately for item names")
            .getBoolean();
        enableVoltageSpecialSearchTooltips = config
            .get(
                "search",
                "enableVoltageSpecialSearchTooltips",
```

Reply:
updated.